dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackerevent/JdbcTrackerEventStore.java

```diff
@@ -784,9 +784,10 @@ private StringBuilder getFromWhereClause(
         .append(
             "inner join trackedentityprogramowner po on (en.trackedentityid=po.trackedentityid and en.programid=po.programid) ")
         .append(
-            "inner join organisationunit evou on (coalesce(po.organisationunitid,"
-                + " ev.organisationunitid)=evou.organisationunitid) ")
-        .append("inner join organisationunit ou on (ev.organisationunitid=ou.organisationunitid) ");
+            "inner join organisationunit ou on (coalesce(po.organisationunitid,"
```

Comment:
Is this `coalesce` there because we were mixing single and tracker events?
Is there a way that a tracker event would not have a program owner?

Reply:
good call, I think a tracker event will always have a program owner, same as enrollments always do, right @ameenhere 
I'll change that